Source/UTF8LineParser.swift

```diff
@@ -77,26 +62,9 @@ class UTF8LineParser {
         return lines
     }
 
-    func closeAndReset() -> [String] {
-        var lines: [String] = []
-
-        if seenCr {
-            lines.append(currentString)
-            currentString = ""
-        }
-
-        if !remainder.isEmpty {
-            currentString.append(replacement)
-            remainder = Data()
-        }
-
-        if !currentString.isEmpty {
-            lines.append(currentString)
-        }
-
-        currentString = ""
+    func closeAndReset() {
```

Comment:
This is simplified because the prior version of the spec dispatched an event on end-of-stream, so we had to handle any remaining data here. Now we can just reset everything.